### PR TITLE
add NN into tutorials

### PR DESCRIPTION
**Pull request by ZJG0**
I have updated the `TUTORIALS_CN.md` and source code. What's more, the code can correctly run.

### CODE REVIEW

File: doc/TUTORIALS_CN.md [hunk 1]
```diff
@@ -869,6 +871,266 @@ Rosetta 完整代码参考 [rtt-ds-lr.py](../example/tutorials/code/rtt-ds-lr.py
...
+
+`tensorflow`版与`Rosetta`版的代码在数据的加载部分是非常不一样的，看完了前面的例子应该很清楚。
+
+```python
+from tensorflow.examples.tutorials.mnist import input_data
+import os
+import tensorflow as tf
+mnist_home = os.path.join("/tmp/data/", 'mnist')
+mnist = input_data.read_data_sets(mnist_home, one_hot=True)
+#将数据分割为训练数据与测试数据
```
Comment (vingstar) [inline doc/TUTORIALS_CN.md:896]: it maybe better not to use Chinese comments

File: doc/TUTORIALS_CN.md [hunk 2]
```diff
@@ -869,6 +871,266 @@ Rosetta 完整代码参考 [rtt-ds-lr.py](../example/tutorials/code/rtt-ds-lr.py
...
+- 加载数据集
+
+这一部分数据集的加载与之前例子中的略有不同，之前加载时在`PrivateDataset`中使用了`load_data`方法，可同时加载特征数据与标签数据。但是，这种同时加载的方式仅支持一维的标签数据，`mnist`中的标签数据是使用`one-hot`编码的共有10维，所以此处使用了加载特征数据的方式加载了标签数据。关于`load_data`、`load_X`、`load_Y`的区别请查阅源代码。
+
+```python
+#加载数据集
+file_x = "./train_x.csv"
+file_y = "./train_y.csv"
+
+X_train = rtt.PrivateDataset(data_owner=(0, 1), label_owner=1).load_X(file_x, header=None)
```
Comment (vingstar) [inline doc/TUTORIALS_CN.md:1072]: For such big dataset, using rtt.PrivateTextLineDataset to load data iteratively may be better.

**Review by vingstar — CHANGES_REQUESTED** [review]
Thanks for contributing!
Some comments have been made, and besides I think we may find a separate repository to host big datasets like the MNIST used in this example.

File: doc/TUTORIALS_CN.md (hunk 1 above)
Comment (ZJG0) [reply doc/TUTORIALS_CN.md:896]: Got it

File: doc/TUTORIALS_CN.md (hunk 2 above)
Comment (ZJG0) [reply doc/TUTORIALS_CN.md:1072]: I tried the API `rtt.PrivateTextLineDataset`, but its return type is not `tensor`,when i call the function`_element_structure()`, the code throws `TypeError: 'TensorStructure' object is not callable`.

Comment (vingstar) [reply doc/TUTORIALS_CN.md:1072]: Yes, this API can NOT used alone, you may check this [sample in tutorial](https://github.com/LatticeX-Foundation/Rosetta/blob/master/doc/TUTORIALS_CN.md#%E6%94%AF%E6%8C%81%E8%B6%85%E5%A4%A7%E6%95%B0%E6%8D%AE%E9%9B%86)

Comment (ZJG0) [reply doc/TUTORIALS_CN.md:1072]: I have read the [sample in tutorial](https://github.com/LatticeX-Foundation/Rosetta/blob/master/doc/TUTORIALS_CN.md#%E6%94%AF%E6%8C%81%E8%B6%85%E5%A4%A7%E6%95%B0%E6%8D%AE%E9%9B%86), but, this sample does not use loss function. In my sample, the loss function requests the parameter whose type is `narray`. However, 
 this API only return `tensor`. Alsoly, this `tensor` can not be converted into `narray`

Comment (vingstar) [reply doc/TUTORIALS_CN.md:1072]: Yeah，I have updated this sample code with TF Optimizer in this [commit](https://github.com/LatticeX-Foundation/Rosetta/commit/17147c41bfd750e7bf67dafc15d787869508870a). You may refer to it now.

Comment (ZJG0) [reply doc/TUTORIALS_CN.md]: I have changed the method of loading data.

File: doc/TUTORIALS_CN.md [hunk 3]
```diff
@@ -869,6 +871,281 @@ Rosetta 完整代码参考 [rtt-ds-lr.py](../example/tutorials/code/rtt-ds-lr.py
...
+执行
+
+```python
+./tutorials.sh rtt mlp_mnist
+```
+
+明文模型会保存在`log/`文件夹下，可以继续执行
+
+```python
+python tf-test_mlp.py
```
Comment (vingstar) [inline doc/TUTORIALS_CN.md:1121]: The name `tf-test_mlp.py`  in this doc is different with the script name `tf-test_mlp_acc.py`. It this a typo?

File: doc/TUTORIALS_CN.md [hunk 4]
```diff
@@ -893,34 +1170,39 @@ dsets/
 │   ├── reg_test_x.csv
 │   ├── reg_test_y.csv
 │   ├── reg_train_x.csv
-│   └── reg_train_y.csv
+│   ├── reg_train_y.csv
+│   ├──	mnist_test_x.csv
```
Comment (vingstar) [inline doc/TUTORIALS_CN.md:1174]: how about aligning them?

File: example/tutorials/code/tf-mlp_mnist.py [hunk 5]
```diff
@@ -0,0 +1,118 @@
+from tensorflow.examples.tutorials.mnist import input_data
+import os
+import tensorflow as tf
+mnist_home = os.path.join("/tmp/data/", 'mnist')
+mnist = input_data.read_data_sets(mnist_home, one_hot=True)
+#将数据分割为训练数据与测试数据
```
Comment (vingstar) [inline example/tutorials/code/tf-mlp_mnist.py:6]: Please also translate these Chinese comments in this code file into English.

**Review by vingstar — CHANGES_REQUESTED** [review]
Generally, LGTM.
I may merge this after some tiny issues as commented in review are fixed.

Besides, I am thinking about moving all the large datasets into another related repository, which can be used as a git submodule in this codebase.  This may need future discussion, and I just log it here.

**Comment by ZJG0** [comment]
PTAL

File: doc/TUTORIALS_CN.md [hunk 6]
```diff
@@ -893,34 +1169,39 @@ dsets/
...
+│   └──	mnist_train_x.csv
 ├── P1
 │   ├── cls_test_x.csv
 │   ├── cls_train_x.csv
 │   ├── reg_test_x.csv
 │   ├── reg_test_y.csv
 │   ├── reg_train_x.csv
-│   └── reg_train_y.csv
+│   ├── reg_train_y.csv
+│   └──	mnist_train_x.csv
```
Comment (vingstar) [inline doc/TUTORIALS_CN.md:1190]: ditto

File: doc/TUTORIALS_CN.md [hunk 7]
```diff
@@ -893,34 +1169,39 @@ dsets/
...
+│   └── mnist_test_y.csv
 ├── P0
 │   ├── cls_test_x.csv
 │   ├── cls_test_y.csv
 │   ├── cls_train_x.csv
 │   ├── cls_train_y.csv
 │   ├── reg_test_x.csv
-│   └── reg_train_x.csv
+│   ├── reg_train_x.csv
+│   └──	mnist_train_x.csv
```
Comment (vingstar) [inline doc/TUTORIALS_CN.md:1182]: this one

File: doc/TUTORIALS_CN.md [hunk 8]
```diff
@@ -16,6 +16,8 @@
     - [模型加载与预测](#模型加载与预测)
   - [逻辑回归](#逻辑回归)
   - [支持超大数据集](#支持超大数据集)
+- [隐私深度学习](#隐私深度学习)
```
Comment (vingstar) [inline doc/TUTORIALS_CN.md:19]: how about also adding in the English version TUTORIALS.md doc?

**Review by vingstar — COMMENTED** [review]
Good.

File: doc/TUTORIALS_CN.md (hunk 6 above)
Comment (ZJG0) [reply doc/TUTORIALS_CN.md:1190]: image
In the commits, there is the extra space, but, not in the document.

File: doc/TUTORIALS_CN.md (hunk 8 above)
Comment (ZJG0) [reply doc/TUTORIALS_CN.md:19]: ok, i will do it.

**Comment by ZJG0** [comment]
PTAL

**Pull request merged**